### DISCUSSION TITLE
Functionality question:  Show Similar Tracks to playlist?

### DISCUSSION THREAD

**Discussion by soilstack** [jpochyla/psst#643] [Ideas]
**Is your feature request related to a problem? Please describe.**
The reason I came to PSST was for the immensely superior 'Show Similar Tracks' functionality.  Its suggestions are generally fresh and fantastic.

I generally listen to Spotify on my phone however, not at my desktop, so I don't have access to these better suggestions.

Is there some way I could perhaps bulk-generate a bunch of suggestions inside PSST and migrate them to a playlist that I'd have access to from Spotify on my phone ?

**Describe the solution you'd like**
Just some way to say, "Dump this suite of suggestions to a new playlist'.  Maybe I'm overlooking a better way that already exists.

**Describe alternatives you've considered**
At present all I've figure out to do is hit the + "save track"  and then go to the Tracks section and individually right-click 'add to playlist' for each one.  Sort of tedious.

**Additional context**
None

**Comment by SO9010** [comment]
That is a great idea, I dont think it should be too hard to save all of the songs into a new playlist! I'll happily work on this. But I want to see if @jacksongoode thinks it will fit in?

**Comment by SO9010** [comment]
I'm wondering if we could make it so we can select multiple songs. The functionality would be so you can click to the right of the tracking widget where the saved icon highlights the track. Then, you can left shift and click to select lots of different songs!
